### DISCUSSION TITLE
Add level mapping to lint rules?

### DISCUSSION THREAD

**Discussion by MurrayC7** [astral-sh/ruff#11098] [Ideas]
For better showing custom levels of different lint rules, could ruff consider add this mapping setting?
I want to straightly know the priority of each rule, so that I can firstly improving important items and put others aside for now.  
Such as {"serious": ["B020", "UP022"], "important": [...], "general": [...]}

**Comment by MichaReiser** [comment]
Hi. Thanks for starting this discussion.

I'm not entirely sure if I understand your ask. Is it that you want to give lint rules different severities (error vs warning, vs info) as outlined in https://github.com/astral-sh/ruff/issues/1256?

**Reply by MurrayC7** [reply]
Yes, that is what i want. Thanks a lot.
